specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json

```diff
@@ -504,6 +504,14 @@
         "ImmediatePurgeDataOn30Days": {
           "type": "boolean",
           "description": "Purge data immediately after 30 days."
+        },
+        "publicNetworkAccessForIngestion": {
+          "title": "The network access type for accessing Application Insights ingestion.",
```

Comment:
should it be description? It's rare to see the field `title` in a defintion.

Reply:
its actually quite common:
see **Microsoft.Batch\stable\2020-03-01\BatchManagement.json**
for example.